ngx-fudis/.storybook/getVersion.js

```diff
@@ -11,14 +12,14 @@ const defaultVersion = {
 };
 
 const getVersion = () => {
-	if (!isValidSemVer(tagOrBranchName)) {
+	if (!isValidSemVer(pkg.version)) {
 		return {
 			...defaultVersion,
 			postfix: tagOrBranchName,
 		};
 	}
 
-	return parseSemVer(tagOrBranchName);
+	return parseSemVer(pkg.version);
```

Comment:
If `pkg.version` is returned, other logic in this file is stale, so should be removed.

However, this change will mean that documentation built from PR branches will be labelled with the latest version which might or might not be what we want. I should note also that this change is not necessary as the build system has control over the label with the current setup (which makes configuring the workflows nice).